force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -182,15 +214,15 @@ public with sharing class FlowControlLoopsCollections {
      * @return A List of integers containing the numbers 1, 2, and 3.
      */
     public static List<Integer> createAndPopulateList() {
+        List<Integer>myNumbers = new List<Integer>();
         // Create a new list of integers
-
-        // Add the integers 1, 2, and 3 to the list
-        // add 1
-        // add 2
-        // add 3
-
-        // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+            // add 1
+            myNumbers.add (1);
+             // add 2
+            myNumbers.add (2);
+            // add 3
+            myNumbers.add (3);
+            return myNumbers;
```

Comment:
Good work.  It will save you effort and produce faster code to learn to use the constructor. 

Here was my solution. 

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
```